### ISSUE TITLE
Wonder where the CuDNNGRU function is defined in the file sent_sim.py

### ISSUE THREAD

**Issue by caidwang** [bojone/margin-softmax#5]
您好, 我下载您的代码尝试跑, 但发现`x = CuDNNGRU(word_size)(x_embedded)`这句报错, 但是浏览整个文件没有找到整个函数是从哪import的或者定义的, 另外,请问您运行这段代码的环境能告知一下么?

**Comment by caidwang** [comment]
the `CuDNNGRU` is a function in `keras.layers`, which is used for accelerating the GRU computation. It can be replaced by `GRU` if the TensorFlow for GPU is not installed. 
The environment of the project is 

> 测试环境的Python版本为2.7，Keras版本为2.1.5，tensorflow后端。

**Comment by bojone** [comment]
如果CuDNNGRU报错，就改为GRU吧
